engine/tests/phpunit/integration/ElggWidgetIntegrationTest.php

```diff
@@ -67,15 +67,368 @@ public function testGetTitle() {
 		]);
 		$this->assertEquals('title_from_definition', $this->widget->getDisplayName());
 	}
-	
-	public function testHandlerRegistration() {
-		$this->assertFalse(elgg_is_widget_type('test_handler'));
-		elgg_unregister_widget_type('test_handler');
 		
-		elgg_register_widget_type(['id' => 'test_handler', 'context' => []]);
-		$this->assertTrue(elgg_is_widget_type('test_handler'));
+	public function testMovingWidgetsWithinSameColumnToLastPosition() {
+		elgg_register_widget_type(['id' => 'widget_move_handler', 'context' => ['move_widget_context']]);
```

Comment:
this remains active during all the following tests???!!!

Reply:
now (un)registering in up/down